Add logger

YodaForce157 opened the pull request:
Creates a simple logger to log to a file.

Matyrobbrt commented:
I can't see any other issues (unless I missed a very obvious one)

Side note: is the new project explorer using the **Theme** of the project?

Commit:
Create logger

Honestly not sure if this is even good but it works, Will eventually need to move all Sys.out to LOGGER.info/debug but will do that when needed. Feel free to do anything to this as it is the simplest logger ever

Commit:
Minor changes
